src/v2/ion/i18nTransformer.js

```diff
@@ -101,6 +100,9 @@ const I18N_OVERRIDE_MAPPINGS = {
   'enroll-authenticator.onprem_mfa.credentials.passcode': 'enroll.onprem.passcode.placeholder',
   'enroll-authenticator.rsa_token.credentials.clientData': 'enroll.onprem.username.placeholder',
   'enroll-authenticator.rsa_token.credentials.passcode': 'enroll.onprem.passcode.placeholder',
+  'enroll-authenticator.symantec_vip.credentials.credentialId': 'enroll.symantecVip.credentialId.placeholder',
```

Comment:
please add the unit test for these mapping

Reply:
Our unit tests do not seem to cover complete parity of the form labels to i18n keys. IMO - the best way to validate the mapping is correct is through `pseudo-loc`, surfacing errors when the English text is shown instead of the mapped key.

@TianGan-okta - are you OK with us deferring testing this logic until @nikhilvenkatraman-okta and team hook up pseudo-loc to our pipeline?

Reply:
Good point, I am fine with it